src/start/AuthScreen.js

```diff
@@ -363,10 +371,6 @@ class AuthScreenInner extends PureComponent<Props> {
   }
 }
 
-const AuthScreen: ComponentType<OuterProps> = connect((state, props) => ({
-  // Not from the Redux state, but it's convenient to validate the URL
-  // in one central place, like here.
-  realm: new URL(props.route.params.serverSettings.realm_uri),
-}))(AuthScreenInner);
+const AuthScreen: ComponentType<OuterProps> = connect((state, props) => ({}))(AuthScreenInner);
```

Comment:
nit: now that the selector isn't doing anything, leave it out -- `connect` notices that and uses it to better optimize by knowing it never needs to call the selector

(See `src/start/{Password,Dev}AuthScreen.js` for the two remaining places we do that. Used to be a lot more before we migrated most components to Hooks.)